src/docs/eviction.dox

```diff
@@ -25,11 +25,34 @@ Eviction has to obtain exclusive access to the page as there may be
 application threads still reading content on the page that is being
 evicted.
 
+Internally eviction uses the following locks:
```

Comment:
Can you describe what shared memory each lock protects?

Reply:
Done